src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,65 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  } = movie;
+
+  return (
+    <li key={imdbId}>
```

Comment:
redundant, you already used key in the MovieCard component when you call MovieList